projects/packages/forms/src/contact-form/css/grunion-admin.css

```diff
@@ -26,6 +26,14 @@
 		padding-left: 8px !important;
 	}
 }
+#export-modal-opener {
```

Comment:
Little surprised to see ID selector instead of class selector but assuming this was simpler change. Not a biggie.